content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc

```diff
@@ -0,0 +1,131 @@
+= Let's OptaPlan your jBPM tasks (part 2) - Moving to Kogito
+wmedvede
+2021-06-15
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: feature, task assignment, integration
+:jbake-social_media_share_image: CreditDisputeProcessDiagram.png
+
+In the previous blog post link:../../../2020/07/03/LetsOptaPlanYourJbpmTasksPart1.html[Let's OptaPlan your jBPM tasks (part 1)] we showed that OptaPlanner can be used for assigning the human tasks produced by a set of business processes to users according to an optimized plan.
```

Comment:
Can we make the first paragraph more answer the question "What is the benefit for me reading this?" What is the value proposition? Why should I care?

Referencing the previous blog is good, but I 'd leave that to the second paragraph, if you want them to read that first to better understand the problem. Readers are lazy, the TLDR of the problem/value should be in the first paragraph.

Readers might not now what Kogito is or what the benefit of kogito is.

Maybe something in the lines of
"In BPMN workflows in the cloud, we can automatically assign tasks to users by combining Kogito processes with OptaPlanner optimization. Furthermore, we can optimize those assignments, to ensure every user has the right skills for the tasks and ..."